blocks/quote/quote.js

```diff
@@ -5,11 +5,7 @@ export default function decorate(block) {
     block.innerHTML = '';
 
     [...grandChildren].forEach((grandChild, index) => {
-      if (index === 0 && grandChild.children.length === 0) {
-        const p = document.createElement('p');
-        p.textContent = grandChild.textContent;
-        grandChild.innerHTML = '';
-        grandChild.appendChild(p);
+      if (index === 0) {
```

Comment:
@jindaliiita as you added the original changes, I assume you'd know best if removing this logic is ok.

Reply:
Yes, i have checked draft pages which i have created at that time, its working fine there.

Reply:
And Visual tests also reported no difference

Reply:
ok, I am a bit confused about why visual tests did not not report any difference. I do see difference between  
https://main--sunstar--hlxsites.hlx.page/sidekick/blocks/quote
https://quote-block-changes--sunstar--hlxsites.hlx.page/sidekick/blocks/quote

Reply:
Change is intentional here but not sure why it has not been reported by the tool.